client/http_sender_test.go

```diff
@@ -78,7 +80,7 @@ func TestHTTPSenderSend(t *testing.T) {
 		t.Fatal(err)
 	}
 	reply := &proto.PutResponse{}
-	sender.Send(Call{Args: testPutReq, Reply: reply})
+	sender.Send(context.TODO(), Call{Args: testPutReq, Reply: reply})
```

Comment:
Tests should probably use context.Background() instead of context.TODO(); only use TODO when there could be a real context that we need to pipe through instead.